packages/web-components/src/divider/divider.options.ts

```diff
@@ -0,0 +1,47 @@
+import { DividerRole, ValuesOf } from '@microsoft/fast-foundation';
+import { Orientation } from '@microsoft/fast-web-utilities';
+
+/**
+ * Fast Foundation DividerRole property
+ * @public
+ */
+export { DividerRole };
+
+/**
+ * Fast Web Utilties Orientation property
+ * @public
+ */
+export { Orientation as DividerOrientation };
```

Comment:
@halerankin Foundation should have a DividerOrientation export with the latest package - can you confirm? This can likely be updated to export that specific update.

Reply:
Yes. I just added it.